### PR TITLE
📝 Add GET Bank Account Balance documentation

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Notion design: https://www.notion.so/centrapay/Get-Bank-Account-Balance-b3f9ba1abff64775877d4a2fb073b123

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -17,9 +17,9 @@ jobs:
       - uses: actions/cache@v1
         with:
           path: legacy/vendor/bundle
-          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          key: ${{ runner.os }}-gems-v2-${{ hashFiles('**/Gemfile.lock') }}
```
Comment (IsaacMurtagh) [inline .github/workflows/publish.yml:20]: Can you add this in a separate PR

File: legacy/src/api/bank-accounts/bank-accounts.md [hunk 2]
```diff
@@ -54,12 +54,13 @@ bank transaction can be used to verify a bank account.
...
-| fullName     | String  | The first and last name of the user.                        |
-| emailAddress | String  | The user's email address.                                   |
-| test         | Boolean | A flag which is present if the Bank Account is for testing. |
+|    Field     |       Type        |                          Description                          |
+| :----------- | :---------------- | :------------------------------------------------------------ |
+| phoneNumber  | String            | The user's phone number.                                      |
+| fullName     | String            | The first and last name of the user.                          |
+| emailAddress | String            | The user's email address.                                     |
+| test         | Boolean           | A flag which is present if the Bank Account is for testing.   |
+| balance      | {% dt Monetary %} | The Bank Account balance, retrieved using Open Banking flows. |
```
[the remark this answers is not in the document]
Comment (IsaacMurtagh) [reply legacy/src/api/bank-accounts/bank-accounts.md:63]: Bank account is its own model

[the remark this answers is not in the document]
Comment (MeganSteenkamp) [reply legacy/src/api/bank-accounts/bank-accounts.md]: @IsaacMurtagh have addressed this, thanks.

**Review by mark-scott — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 1 above)
Comment (MeganSteenkamp) [reply .github/workflows/publish.yml]: Done @IsaacMurtagh. Can you please review the PR again?

**Review by IsaacMurtagh — APPROVED** [review]
Please fix bank account balance model before approving

**Pull request merged**